PowerUp/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="black">#000000</color>
+    <color name="purple_light">#9b7adc</color>
```

Comment:
This is fine, but I would rename the colors including the name `powerup` since pink, yellow, etc are already defined in android and it can get confusing